{{cookiecutter.app_name}}/Pipfile

```diff
@@ -12,6 +12,9 @@ click = ">=5.0"
 # Database
 Flask-SQLAlchemy = "==2.4.0"
 SQLAlchemy = "==1.3.8"
+{%- if cookiecutter.deployment_on_heroku == "yes" %}
+psycopg2 = "==2.8.3"
```

Comment:
Could you add a comment for why this package is only necessary for Heroku deployments?

Reply:
`psycopg2` is necessary to support PosgreSQLl (IMO the best database for flask on Heroku. SQLite db will be lost at least once every 24 hours due to ephemeral filesystem of Heroku. source: https://devcenter.heroku.com/articles/sqlite3#disk-backed-storage).
@jamescurtin did you mean comment here or in code?

Reply:
This makes sense! No need to add anything to the code.

One followup thought, though, on the usage of `psycopg2`: It has a number of [build prerequisites](http://initd.org/psycopg/docs/install.html#build-prerequisites), that I suspect may cause confusion for new users who don't have a development environment set up (C compiler, multiple system binaries, possibly needing to modify $PATH, etc.)

The user-friendly alternative would be to use `psycopg2-binary`, though it's documentation states 
> The binary package is a practical choice for development and testing but in production it is advised to use the package built from sources.

What are your thoughts? I would default to using `psycopg2-binary` to avoid situations where a new user gets a confusing pip install issue and gives up, but add a note in the Postgres section of the docs with a link to the `psycopg2` reference guide so that users who will deploy to production can build from source.

Reply:
I think using `psycopg2-binary`  is a good choice because it works everywhere (locally, in docker, on Heroku).
I will set `psycopg2-binary` in the requirements and add explicit information that it should be revised for production.